welcome/index.adoc

```diff
@@ -61,13 +61,27 @@ does not have full access to the internet, you can
 xref:../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[mirror the {product-title} installation images]
 and install a cluster in a restricted network.
 
-- **xref:../installing/installing-gather-logs.adoc#[Check installation logs]**: Access installation logs to evaluate issues that occur during {product-title} {product-version} installation.
+- **Install a cluster in an existing network**: If you use an existing VPC in
```

Comment:
Do we want to define VPC? I had to look, hilariously, but it's _Virtual Private Cloud_.